PostgresV11.md

```diff
@@ -0,0 +1,98 @@
+### V11 postgres migration steps
+
+##  Pull latest ccd docker
+* Synch all micro-services with the same branch version for instance: develop
+* Start ccd docker and make sure that the new container ccd-shared-database-v11 is up and running.
+* uncomment ccd-shared-database-v11 section in the backend.yml
+````
+#This section should be commented when the pr is merged in master.
+  ccd-shared-database-v11:
+    build: ../database-v11
+    healthcheck:
+      interval: 10s
+      timeout: 10s
+      retries: 10
+    environment:
+      DB_USERNAME:
+      DB_PASSWORD:
+      POSTGRES_HOST_AUTH_METHOD: trust
+    ports:
+      - 5055:5432
+    volumes:
+      - ccd-docker-ccd-shared-database-data-v11:/var/lib/postgresql/data
+    networks:
+      - ccd-network
+
+````
+
+##  Backup all DB containers
+* get your container id for instance: a210d7e11a5b
+```
+docker ps | grep compose_ccd-shared-database_1
+```
+* Backup all Dbs in a dumpfile file
+```
+docker exec -it a210d7e11a5b  /usr/bin/pg_dumpall -U  postgres  > dumpfile
+```
+* get your container id of the V11 DB for instance: 36a8eb5cccba
+```
+docker ps | grep compose_ccd-shared-database_1-V11
+````
+* copy the dumpfile to the V11 container DB
+```
+docker cp dumpfile 36a8eb5cccba:/home
+```
+* Import dumpfile in to your new DB V11
+```
+bash the container
+docker exec -it 36a8eb5cccba bash
+
+change permission to your dumpfile file
```

Comment:
are you missing the `*` here and in some of the lines below?

Reply:
DONE